package/com.unity.formats.usd/Runtime/Scripts/IO/Geometry/MeshImporter.cs

```diff
@@ -264,7 +264,8 @@ public class GeometrySubsets {
       // rely on the UnityEngine.Mesh API.
 
       Material mat = renderer.sharedMaterial;
-      bool changeHandedness = options.changeHandedness == BasisTransformation.SlowAndSafe;
+      bool changeHandedness = options.changeHandedness == BasisTransformation.SlowAndSafe ||
+                              options.changeHandedness == BasisTransformation.SlowAndSafeAsFBX;
```

Comment:
That is comforting ^^, thanks for the spot!

Reply:
+1, nice catch!